### PR TITLE
chore: add script to update package-level package-lock.json

### PR DESCRIPTION
**Pull request by raymondfeng**
This PR adds a npm script to update package-level package-lock.json files as `lerna bootstrap` does not do so. See https://github.com/lerna/lerna/issues/1415#issuecomment-455790478 for more details.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,10 +26,11 @@
     "postinstall": "lerna bootstrap && npm run -s update-greenkeeper-json",
     "prerelease": "npm run build:full && npm run mocha && npm run lint",
     "release": "lerna version && lerna publish from-git --yes",
+    "update-package-lock-json": "lerna exec \"npm install -s --ignore-scripts --package-lock-only --no-audit\"",
     "update-template-deps": "node bin/update-template-deps -f",
     "update-greenkeeper-json": "node bin/update-greenkeeper-json -f",
     "sync-dev-deps": "node bin/sync-dev-deps",
-    "version": "npm run update-template-deps && npm run apidocs",
+    "version": "npm run update-package-lock-json && npm run update-template-deps && npm run apidocs",
```
Comment (b-admike) [inline package.json]: Does running `update-template-deps` before or after `update-package-lock-json` make any difference to the `package-lock.json` files that are updated?

File: package.json [hunk 2]
```diff
@@ -26,10 +26,11 @@
     "postinstall": "lerna bootstrap && npm run -s update-greenkeeper-json",
     "prerelease": "npm run build:full && npm run mocha && npm run lint",
     "release": "lerna version && lerna publish from-git --yes",
+    "update-package-lock-json": "lerna exec \"npm install -s --ignore-scripts --package-lock-only --no-audit\"",
```
Comment (b-admike) [inline package.json]: Can we add `git add packages/*/package-lock.json"` at the end of the script as pointed out in https://github.com/lerna/lerna/issues/1415#issuecomment-455790478?

File: packages/build/package.json [hunk 3]
```diff
@@ -43,8 +43,5 @@
   "scripts": {
     "test": "npm run mocha",
     "mocha": "node bin/run-mocha --timeout 30000 \"test/integration/*.js\""
-  },
-  "devDependencies": {
-    "@loopback/tslint-config": "^2.0.0"
```
Comment (nabdelgadir) [inline packages/build/package.json]: Just curious, why was this removed?

Comment (raymondfeng) [reply packages/build/package.json]: It's already listed in `dependencies`.

File: package.json (hunk 2 above)
Comment (raymondfeng) [reply package.json]: Good catch. I forgot that.

File: package.json (hunk 1 above)
Comment (raymondfeng) [reply package.json]: Not really.

File: package.json (hunk 2 above)
Comment (raymondfeng) [reply package.json]: Fixed.

File: benchmark/package-lock.json [hunk 4]
```diff
@@ -1,2704 +1,2798 @@
...
-		"path-is-absolute": {
-			"version": "1.0.1",
-			"resolved": "https://registry.npmjs.org/path-is-absolute/-/path-is-absolute-1.0.1.tgz",
-			"integrity": "sha1-F0uSaHNVNP+8es5r9TpanhtcX18=",
-			"dev": true
-		},
-		"path-key": {
-			"version": "2.0.1",
-			"resolved": "https://registry.npmjs.org/path-key/-/path-key-2.0.1.tgz",
-			"integrity": "sha1-QRyttXTFoUDTpLGRDUDYDMn0C0A=
```
Comment (bajtos) [inline benchmark/package-lock.json]: I think this is not right. Package lock should not list local dependencies that are linked within monorepo.

In the current setup, `npm i` (`lerna bootstrap`) installs monorepo-local packages from the npm registry and replaces any links created by `lerna`.

**Comment by bajtos** [comment]
@raymondfeng I feel this change is very likely going in a wrong direction. Why is it necessary to update package-lock files when we are making new releases? The lock files are not supposed to contain entries for dependencies resolved via symlinks by lerna.

I am fixing the immediate problem in https://github.com/strongloop/loopback-next/pull/2593.

I think we should also revert the changes made in `package.json` by this pull request. Thoughts?

File: benchmark/package-lock.json (hunk 4 above)
Comment (raymondfeng) [reply benchmark/package-lock.json]: For the record, `npm i` does not update package-lock.json for me even after I added new dependencies to a package.json. That's what led me to discover the issue and put a workaround.

Comment (raymondfeng) [reply benchmark/package-lock.json]: It seems that `lerna clean && lerna bootstrap --no-ci` does update package-lock.json for all leaf packages. I'll open a PR to use it.

**Comment by raymondfeng** [comment]
> I feel this change is very likely going in a wrong direction. Why is it necessary to update package-lock files when we are making new releases? The lock files are not supposed to contain entries for dependencies resolved via symlinks by lerna.

I wanted to make sure package-lock.json files are up-to-date respecting package.json. For example, if I add a new dependency to `packages/rest/package.json`, `npm i` at `loopback-next` level does not automatically update package-lock.json and it fails CI as CI uses `npm ci` and the new dep will be missing.
